README.md

```diff
@@ -4,6 +4,12 @@
 
 📖 Deploy docs from your source tree to the GitHub wiki
 
+<div align="center">
+
+![](https://user-images.githubusercontent.com/61068799/231881220-2915f956-dbdb-4eee-8807-4eba9537523f.png)
+
+</div>
```

Comment:
```suggestion
![](https://user-images.githubusercontent.com/61068799/231881220-2915f956-dbdb-4eee-8807-4eba9537523f.png)
```
Thoughts on just aligning this left? IMO it felt a bit awkward when my focus changed from left-to-right starting at the top of the README to the horizontal center, then back to left-to-right.

Reply:
mostly just because that's the most common user-expected way images are shown on almost every website:

![image](https://user-images.githubusercontent.com/61068799/232905302-a8d4afa6-6f06-43c8-b1fe-5999f6705847.png)
![image](https://user-images.githubusercontent.com/61068799/232905692-e982bf1b-2d89-4881-9c9c-497a7ce1fb88.png)
![image](https://user-images.githubusercontent.com/61068799/232905715-318b61b1-7f15-45da-99d9-40f0d3bde41f.png)
![image](https://user-images.githubusercontent.com/61068799/232905817-b4e0a508-5f96-4bca-8908-86ab61eb7cec.png)
![image](https://user-images.githubusercontent.com/61068799/232905889-cb0e7924-8f0d-4b0c-9978-23f964dac228.png)
![image](https://user-images.githubusercontent.com/61068799/232905995-fd05cff8-35be-428f-ba1d-00df63440547.png)


so i guess it's kinda the norm?
also here's a (cherry picked) example of a site recommending centering images: https://artsassistance.com/centering-dos-and-donts-for-blog-posts-and-web-pages/

Reply:
Alright, sounds good.

Reply:
also its worth noting that yes, it does look bad when the text is like 1500px wide

![image](https://user-images.githubusercontent.com/61068799/232930484-067d39fd-8842-4efa-a4d5-dc7941ba9322.png)


but it looks better in the default view: https://github.com/jcbhmr/actions-wiki/tree/patch-5#readme

![image](https://user-images.githubusercontent.com/61068799/232930442-e22743b0-b112-420d-92e2-f9dd2f275efb.png)